nearai/cli.py

```diff
@@ -577,7 +577,11 @@ def interactive(
 
         last_message_id = None
         print(f"\n=== Starting interactive session with agent: {agent_id} ===")
-        print("Type 'exit' to end the session\n")
+        print("")
+        print("Type 'exit' to end the session")
+        print("On Linux/macOS: Press Ctrl+D at the beginning of a new line after the prompt")
```

Comment:
```suggestion
        print("On Linux/macOS: To submit, press Ctrl+D at the beginning of a new line after your prompt")
        print("Type 'exit' to end the session")
```

Based on the above conversation, I would phrase it this way. Would it be possible to just have it submit by pressing Ctrl+D at any time rather on a new line & at the beginning?

Reply:
Sending EOF is OS-specific. I don't think I can change it.